tests/test_tracing.py

```diff
@@ -647,6 +648,33 @@ def test_step_function_trace_data(self):
             expected_context,
         )
 
+    def test_is_legacy_lambda_step_function(self):
+        sqs_event = {
```

Comment:
I know it doesn't matter much, but do we want to use the name `sqs_event` when we've written this for Step Functions?

Reply:
Good point, I just copied this over from another test

Will update this